content/20.pharmaceuticals.md

```diff
@@ -0,0 +1,392 @@
+Broad-spectrum treatments are not specifically targeted at a virus or at particular host receptors, but rather induce broad shifts in host biology that are hypothesized to be potential inhibitors of the virus.
+Instead, when a virus enters a host, the host becomes the virus's environment, and, therefore, the state of the host can also influence the virus's ability to replicate and spread.
+In most cases, interest in particular candidate medications arises because they are already available for other purposes and clinicians are able to administer them to patients quickly.
+However, the fact that the targets of these agents are non-specific means that the mechanism of action can appear to be relevant to COVID-19 without a therapeutic or prophylactic effect being observed in clinical trials.
+Over the last year, dexamethasone has emerged as the strongest treatment against severe COVID-19, but ARBs/ACEis have not borne out early hopes in their potential.
+Given that it is used to treat ARDS, the clinical relevance of this drug to the COVID-19 symptomatology is clear.
+The dexamethasone story clearly illustrates how clinical context can be a valuable approach through which to identify candidate therapeutics.
+
+An alternative strategy in the COVID-19 pandemic has been to identify candidate pharmaceuticals based on information gained in research on prior HCoV. 
+These approaches often involve biologics that have been developed to imitate the function of endogenous compounds.
+To date, biologics have been explored that imitate IL-6 and <!--To Do: finish summary-->
+Once again, this approach does not require the development of compounds specifically designed for the emergent viral threat.
+However, once again, these compounds have shown varying success.
+<!--To Do: Summarize 2 & 3-->
+
+Finally, computational approaches can allow for the rapid screening of a large number of compounds to identify those most likely to display a desired behavior or function.
+However, this approach focuses on modeling interactions between a compound and a virus rather than their observed interactions.
+These compounds also may lack some of the benefits of other categories; for example, computationally designed compounds typically lack safety information, whereas for more established therapeutics,  a general understanding of the drug's safety in other contexts may be available prior to the beginning of clinical trials.
+
+The all-hands-on-deck approach that the scientific community has taken towards the COVID-19 pandemic has facilitated an unprecedented breadth of research into an emerging viral threat.
+Experience gained through prior outbreaks of HCoV combined with recent scientific advances in other fields ranging from <!--To Do: OK I can actually only think of examples from vaccines?--> to high-performance computing have enabled the rapid assessment of candidate drugs using a number of different approaches.
```

Comment:
Is it better to emphasize the use of theraputics in combination with vaccines to help control the spread and severity of diease?

Reply:
See below!